.github/workflows/tests.yaml

```diff
@@ -46,70 +59,93 @@ jobs:
           make test-pylint
       - name: Run pyright
         run: |
-          sudo snap install --classic node
-          sudo snap install --classic pyright
           make test-pyright
-      - name: Install LXD
+
+  unit-tests:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
Do we need to test on 3.9? I'd kinda prefer to do 3.8, 3.10, 3.11 if possible, getting us a [focal, jammy, latest] support matrix.

Reply:
I removed 3.9 and created an issue for 3.11 support.